src/App.jsx

```diff
@@ -9,30 +12,41 @@ import Login from './components/login';
 import * as GitHubService from './services/github';
 import * as DocumentService from './services/document';
 import events from './services/events';
+import store from './store';
+import SchemaEditor from './components/schema-editor';
 
 injectGlobal`
   * { box-sizing: border-box; }
   body { margin: 0; }
 `;
 
+const mapStatetoProps = state => ({
+  schema: state.schema,
+  content: state.content,
+});
+
+events.on('commit', () => {
+  store.dispatch({ type: 'SET_CONTENT', value: DocumentService.getJSON() });
+});
+
+const WrappedDocumentViewer = connect(mapStatetoProps)(DocumentViewer);
```

Comment:
I usually `connect` the component when exporting it, this makes the component more flexible, but means you have to wire it up every time you use it.